libs/pika/async_cuda_base/include/pika/async_cuda_base/cuda_stream.hpp

```diff
@@ -15,12 +15,19 @@
 #include <string>
 
 namespace pika::cuda::experimental {
-    /// RAII wrapper for a CUDA stream.
+    /// \brief RAII wrapper for a CUDA stream.
     ///
-    /// An RAII wrapper for a CUDA stream which creates a stream on construction
-    /// and destroys it on destruction. Is movable and copiable. A moved-from
-    /// stream holds the default stream. A copied stream uses the properties
-    /// from the given stream and creates a new stream.
+    /// An RAII wrapper for a CUDA stream which creates a stream on construction and destroys it on
+    /// destruction. Is movable and copyable. A moved-from stream holds the default stream. A copied
+    /// stream uses the properties from the given stream and creates a new stream.
+    ///
+    /// Equality comparable and formattable.
+    ///
+    /// When accessing the underlying stream, https://github.com/eth-cscs/whip is used for
```

Comment:
This will already be rendered as a clickable link in the final documentation, but do you mean something like `[whip](https://github.com/eth-cscs/whip)`?

Reply:
Either is fine, I forgot links are automatically clickable.

Reply:
I changed this to a markdown link by the way. It looks nicer in the text.